src/app/[activityId]/_components/TabletReserveModal.tsx

```diff
@@ -0,0 +1,257 @@
+'use client';
+
+import { useEffect, useRef, useState } from 'react';
+
+import activitiesApi from '@/api/activitiesApi';
+import { useActivityDetail } from '@/app/[activityId]/_hooks/queries/useActivityDetail';
+import { useAvailableSchedule } from '@/app/[activityId]/_hooks/queries/useAvailableSchedule';
+import Button from '@/components/Button';
+import Icon from '@/components/Icon';
+import { useClickOutside } from '@/hooks/useClickOutside';
+import { useUserStore } from '@/stores/userStore';
+import { getCalendarDates } from '@/utils/dateUtils';
+
+const isSameMonth = (base: Date, target: Date) =>
+  base.getFullYear() === target.getFullYear() &&
+  base.getMonth() === target.getMonth();
+
+interface TabletReserveModalProps {
+  activityId: number;
+  onClose: () => void;
+  onReserved: () => void;
+}
+
+export default function TabletReserveModal({
+  activityId,
+  onClose,
+  onReserved,
+}: TabletReserveModalProps) {
+  const modalRef = useRef<HTMLDivElement>(null);
+  useClickOutside(modalRef, onClose);
+
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [selectedDate, setSelectedDate] = useState<string | null>(null);
+  const [selectedTimeId, setSelectedTimeId] = useState<number | null>(null);
+  const [peopleCount, setPeopleCount] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const year = String(currentDate.getFullYear());
+  const month = String(currentDate.getMonth() + 1).padStart(2, '0');
+  const dates = getCalendarDates(currentDate);
+
+  const { data: detail } = useActivityDetail(activityId);
+  const { data: schedules = [] } = useAvailableSchedule(
+    activityId,
+    year,
+    month,
+  );
+
+  const user = useUserStore((state) => state.user);
+  const isMyActivity = user && detail?.userId === user.id;
+
+  const price = detail?.price ?? null;
+  const totalPrice = price ? price * peopleCount : null;
+
+  const timeOptions = selectedDate
+    ? (schedules.find((s) => s.date === selectedDate)?.times ?? [])
+    : [];
+
+  const handleReserve = async () => {
+    if (!selectedTimeId) return;
+    try {
+      setIsLoading(true);
+      await activitiesApi.postReservation(activityId, {
+        scheduleId: selectedTimeId,
+        headCount: peopleCount,
+      });
+      onReserved();
+    } catch (error) {
+      alert('예약에 실패했습니다.');
+      console.error(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    const originalStyle = window.getComputedStyle(document.body).overflow;
+    document.body.style.overflow = 'hidden';
+    return () => {
+      document.body.style.overflow = originalStyle;
+    };
+  }, []);
+
+  if (isMyActivity) return null;
+
+  return (
+    <div className='fixed inset-0 z-50 flex items-end justify-center bg-black/40'>
+      <div
+        ref={modalRef}
+        className='max-h-[80vh] w-full overflow-y-auto rounded-t-2xl bg-white p-20 md:w-full md:rounded-2xl'
```

Comment:
```suggestion
        className='max-h-[80vh] w-full overflow-y-auto rounded-t-2xl bg-white p-20'
```
❗ tablet일 때만 보이는데 `md` breakpoint를 설정할 이유는 없어보입니다~